### ISSUE TITLE
GitHub Actions Migration Process Proposal

### ISSUE THREAD

**Issue by Aveline-art** [hackforla/engineering#34]
### Overview
In order to migration our GitHub Actions from a repo level to a org level, we need to figure out how this process will work. For this issue, we will create, review, and approve a proposal for this migration.

### Action Items

- [ ] Create proposal
- [ ] Discuss proposal
- [ ] Approve proposal

**Comment by Aveline-art** [comment]
# GitHub Actions Migration Process Proposal

## Process Steps

###

**Comment by ExperimentsInHonesty** [comment]
@Aveline-art do you have any changes/updates to the proposal based on the actions we took last time we meet?
- renamed the https://github.com/hackforla/.github repo to https://github.com/hackforla/.github-hackforla-base-repo-template
- created a new https://github.com/hackforla/.github

**Comment by Aveline-art** [comment]
@ExperimentsInHonesty Not really. The usage of the `.github` repo was within what was expected by the proposal.

I did make the `.github` [repo](https://github.com/hackforla/.github), with all the support files that are integrated with Github as well as a demo of workflow templates.
